src/style/mixins.scss

```diff
@@ -46,3 +46,46 @@
 @mixin transition-bg-color {
   transition: background-color 200ms ease-in-out;
 }
+
+@mixin twelve-column-layout {
+  row-gap: 40px;
```

Comment:
We may want no gaps by default ? I don't see 40px as a standard row gap on the Figma.